test/core/componentTests.ts

```diff
@@ -203,13 +203,13 @@ describe("Component behavior", () => {
   it("clipPath works as expected", () => {
     assert.isFalse(c.clipPathEnabled, "clipPathEnabled defaults to false");
     c.clipPathEnabled = true;
-    var expectedClipPathID = c.getID();
     c.anchor(svg);
     c.computeLayout({ x: 0, y: 0 }, 100, 100);
     c.render();
+    var expectedClipPathID = (<any>c)._boxContainer[0][0].firstChild.id;
```

Comment:
it is expected. it is taken from another component. The current component is `expected` to have the same id as the other component